apps/api/src/app/workflows-v2/shared/schemas/delay-control.schema.ts

```diff
@@ -26,15 +26,15 @@ export const delayUiSchema: UiSchema = {
   properties: {
     amount: {
       component: UiComponentEnum.DELAY_AMOUNT,
-      placeholder: '30',
```

Comment:
We probably can't have it like this, see [this thread](https://novu.slack.com/archives/C07HVLG7G2U/p1732895403877039?thread_ts=1732887674.583079&cid=C07HVLG7G2U) for more info.